.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+	"useTabs": true,
```

Comment:
This does against our JavaScript standards 😜
https://github.com/eventbrite/javascript

Reply:
Also, could you add `"singleQuote": true,`

Reply:
Well, I'm off to change the JavaScript standards, cause tabs are objectively better than spaces, that's not even meme-ing, it's just factual.